### DISCUSSION TITLE
Thumbs center in Carousel

### DISCUSSION THREAD

**Discussion by Baalamurgan** [leandrowd/react-responsive-carousel#620] [Q&A]
How to center thumbs?

**Comment by devsakae** [comment]
If you are using TailwindCSS, just add the class 'text-center' to Carousel className, like this:

```
<Carousel className='text-center'>
...
```

In pure CSS, create a class with `text-align: center;`

**Reply by ggsheeet** [reply]
Thank you so much for this!

**Reply by ASLAM-stack** [reply]
Thanks, you so mush
